find_white_in_terminal.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import sys
+import re
+
+
+in_prompt = False
+
+with open(sys.argv[1]) as infile:
+    for lineno, line in enumerate(infile.read().splitlines()):
+        if '\\begin{prompt}' in line:
+            in_prompt = True
+        elif '\\end{prompt}' in line:
+            in_prompt = False
+        elif '%' in line and in_prompt:
+            # Find latex commands between % %
+            latex_interpreted = re.findall(r'%([^%]*)%', line)
+            for match in latex_interpreted:
+                # Find latex command not ending in {.
+                result = re.search(r'\\[a-zA-Z]+($|[^a-zA-Z{])', match)
```

Comment:
Use `\w` rather than `[a-zA-Z]`?

Also, you want to find anything that (starts with `\\` and) doesn't end with `{}`, right?

Would this work? Not sure if allowing whitespace makes sense, is `%  \\example %` valid LaTeX or *must* it be `%\\example%` w.r.t. whitespace?
```python
result = re.search(r'\\\w+\s*$')